ports/esp32/mphalport.c

```diff
@@ -122,9 +114,7 @@ void mp_hal_stdout_tx_strn(const char *str, size_t len) {
     #elif CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG
     usb_serial_jtag_tx_strn(str, len);
     #else
-    for (uint32_t i = 0; i < len; ++i) {
-        uart_tx_one_char(str[i]);
-    }
+    uart_repl_write(str, len);
```

Comment:
I'd call this function `uart_tx_strn()` to match the others here.

Reply:
I was actually thinking of renaming the others to be honest, they're named like a generic uart function but in actuality they're hardcoded to repl / uart0 use

Reply:
I meant to match `usb_tx_strn` and `usb_serial_jtag_tx_strn` (which are generic and not tied to the REPL).

Or how about `uart_stdout_tx_strn()`?  Or `uart_tx_strn(MICROPY_HW_UART_REPL, ...)`?

Reply:
Ah yes I follow now, I've just updated as such